src/lib/components/explorer/export/ExportStepper.svelte

```diff
@@ -0,0 +1,285 @@
+<script lang="ts">
+  import * as api from '$lib/api';
+  import { browser } from '$app/environment';
+  import Stepper from '$lib/components/steppers/horizontal/Stepper.svelte';
+  import Step from '$lib/components/steppers/horizontal/Step.svelte';
+  import Datatable from '$lib/components/datatable/Table.svelte';
+  import Summary from './Summary.svelte';
+  import UserToken from '$lib/components/UserToken.svelte';
+  import CopyButton from '$lib/components/buttons/CopyButton.svelte';
+  import type { ExportRowInterface } from '$lib/models/ExportRow';
+  import type { QueryRequestInterface } from '$lib/models/api/Request';
+  import { ProgressRadial } from '@skeletonlabs/skeleton';
+  import ErrorAlert from '$lib/components/ErrorAlert.svelte';
+  import ExportStore from '$lib/stores/Export';
+  let { exports } = ExportStore;
+  import { state } from '$lib/stores/Stepper';
+  import { goto } from '$app/navigation';
+  import { type DataSet, type DatasetError } from '$lib/models/Dataset';
+  import { createDatasetName } from '$lib/services/datasets';
+
+  export let query: QueryRequestInterface;
+  export let showTreeStep = false;
+  export let rows: ExportRowInterface[] = [];
+  let statusPromise: Promise<string>;
+  let preparePromise: Promise<void>;
+  let datasetNameInput: string = '';
+  let picsureResultId: string = '';
+  let lockDownload = true;
+  let error: string = '';
+  let namedDataset: DataSet;
+  $: datasetId = '';
+  $: canDownload = true;
+  $: apiExport = false;
```

Comment:
What sets this?

Reply:
Its supposed to be the features from .env but We have not gotten that far yet. They don't need to be reactive either.